public/app/plugins/panel/candlestick/fields.ts

```diff
@@ -184,11 +184,15 @@ export function prepareCandlestickFields(
   // so they fall through to unmapped fields and get appropriate includeAllFields treatment
   if (options.mode === VizDisplayMode.Volume) {
     if (data.high) {
-      used.delete(data.high);
+      if (data.high !== data.open) {
+        used.delete(data.high);
+      }
       data.high = undefined;
     }
     if (data.low) {
-      used.delete(data.low);
+      if (data.low !== data.open) {
+        used.delete(data.low);
+      }
       data.low = undefined;
     }
```

Comment:
So this function has unit tests, the nice thing with that is now that we identified a bug/crash/new case we can just add a new scenario that triggers the error, then add the code fixes it :) 
https://github.com/grafana/grafana/blob/f858b483c634cbf54f9683775b2edef377aa08fc/public/app/plugins/panel/candlestick/fields.test.ts#L57

Reply:
added some tests